src/Template/Plates.php

```diff
@@ -15,7 +15,7 @@
 /**
  * Template implementation bridging league/plates
  */
-class Plates implements TemplateInterface
+class Plates implements TemplateRendererInterface
```

Comment:
Similarily to Route, I think we should suffix by Renderer (PlatesRendere, ZendRenderer, TwigRenderer...)